src/sql/code_generator/ob_static_engine_cg.cpp

```diff
@@ -2733,7 +2733,7 @@ int ObStaticEngineCG::generate_spec(ObLogGranuleIterator &op, ObGranuleIteratorS
       "desc", op.desc_order(),
       "flags", op.get_gi_flags(),
       "tsc_ids", spec.pw_dml_tsc_ids_,
-      "repart_prunint_idx", spec.repart_pruning_tsc_idx_,
+      "repart_pruning_idx", spec.repart_pruning_tsc_idx_,
```

Comment:
'repart_pruning_tsc_idx" would be better， the original "pruning" means "pruning_tsc"